dockerfile-image-update/src/test/java/com/salesforce/dockerfileimageupdate/utils/DockerfileGitHubUtilTest.java

```diff
@@ -404,25 +404,36 @@ public void testFindImagesAndFix_doNotDeleteOtherLines() throws Exception {
         assertEquals(strB.toString(), "hello\nFROM image:newtag\nthis is a test\n\n\n\nworld\n");
     }
 
-    @Test
-    public void testFindImagesAndFix_doNotDeletePostTagData() throws Exception {
+    @DataProvider
+    public Object[][] postTagData() {
```

Comment:
Wouldn't it be better to do this kind of testing directly against `getEndOfTagLength(String)` to keep the locus of test smaller and narrow scope (smaller unit for the unit test)?